mockOSApi.v2/Models/OSObject.cs

```diff
@@ -2,54 +2,62 @@
 
 namespace mockOSApi.Models;
 
-public abstract class OSObject {
+public abstract class OSObject
+{
 
 
     /* Fields and properties
     *******************************
     */
     private int _handle;
-    public int Handle {
-            get => _handle;
-            set => _handle = value;
-        }
-    public int HashCode { get; set;
-        }
+    public int Handle
+    {
+        get => _handle;
+        set => _handle = value;
+    }
+    public int HashCode
+    {
+        get; set;
+    }
 
     /*
     Constructor members
     ********************************
     */
 
-    public OSObject() {
-            SetHashCode();
-        }
+    public OSObject()
+    {
+        SetHashCode();
+    }
 
     /*
     Methods
     ********************************
     */
 
-    public override string ToString() {
-            return $"{GetType().ToString()}" + " object";
-        }
-    
+    public override string ToString()
+    {
+        return $"{GetType().ToString()}" + " object";
+    }
+
     public override int GetHashCode()
-        {
-           return this.HashCode;  
+    {
+        return this.HashCode;
 
+    }
+
+
+    private void SetHashCode()
+    {
+
+        byte[] source = new byte[14];
```

Comment:
So generally, the HashCode is based solely on the data inside the object.
Two objects that have the exact same values, should have the same HashCode.
If you want to make use of HashCode, start using [C# Records](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/record).
They are a specialized class that implements by default a constructor, equals and HashCode functions

Reply:
removed SetHashCode as will not need it in future